src/pages/Main/components/Sidebar/components/GroupPreview.js

```diff
@@ -14,6 +15,15 @@ const GroupPreview = (props) => {
         changeCurrentGroup(group.id, group.groupName);
     };
 
+    const maxMessageLength = 20;
+    let displayMessage = '';
+
+    if (message.length > maxMessageLength) {
+        displayMessage = message.slice(maxMessageLength) + '...';
```

Comment:
you can use displayMessage = message.length > maxMessageLength ? message.slice(maxMessageLength) + '...'  : message here